src/weakptr.c

```diff
@@ -796,13 +882,19 @@ void SaveWPObj( Obj wpobj )
 void LoadWPObj( Obj wpobj )
 {
   UInt len, i;
+#ifndef USE_JULIA_GC
   Obj *ptr;
   ptr = ADDR_OBJ(wpobj)+1;
+#endif
   len =   LoadUInt();
   STORE_LEN_WPOBJ(wpobj, len);
   for (i = 1; i <= len; i++)
     {
+#ifdef USE_JULIA_GC
+      SET_ELM_WPOBJ(wpobj, i, LoadSubObj());
+#else
```

Comment:
Very minor suggestion, I think we could just use the Julia codepath here -- it would be valid for GASMAN (I'm 99% sure). It might be slightly less efficient, but would save code seperation.

Reply:
Fine by me, will change it.